AdminUi/src/AdminUi/Controllers/TiersController.cs

```diff
@@ -40,6 +41,14 @@ public async Task<IActionResult> GetTiersAsync([FromQuery] PaginationFilter pagi
         return Paged(tiers);
     }
 
+    [HttpGet("{tierId}")]
+    [ProducesResponseType(typeof(GetTierByIdResponse), StatusCodes.Status200OK)]
```

Comment:
Always remember to add further `ProducesResponseType` attributes, depending on what can actually be returned. So in this case, a 404 is possible as well.